<launch>
     <node pkg="topp" type="main_node" name="main_node" output="screen">
        <param name="wheelbase_length"                  value="1.00" />
        <param name="v_max"                             value="5.0"/>    
        <param name="a_max"                             value="1.0"/> 
         <param name="obs_num"                             value="11.0"/>                       
    </node>
     <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find topp)/launch/rviz_config/rviz_test0.rviz">
    </node>

</launch>